testnets/cascadiatestnet/chain.json

```diff
@@ -0,0 +1,126 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "cascadiatestnet",
+  "status": "live",
+  "network_type": "testnet",
+  "website": "https://www.cascadia.foundation/",
+  "pretty_name": "Cascadia",
+  "chain_id": "cascadia_6102-1",
+  "bech32_prefix": "cascadia",
+  "daemon_name": "cascadiad",
+  "node_home": "$HOME/.cascadiad",
+  "key_algos": [
+    "ethsecp256k1"
+  ],
+  "slip44": 60,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "aCC",
```

Comment:
the fee token must be the base unit, but it looks like aCC is the 18 exponent unit. Should this be TCC?

Reply:
fixed above